packages/node_modules/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -313,7 +340,7 @@ skipInBrowser(describe)('plugin-meetings', () => {
               assert.calledWith(initialSetup, {url: url1});
             });
           });
-          describe('destory non active meeting', () => {
+          xdescribe('destory non active meeting', () => {
```

Comment:
why are all these things disabled? (multiple xdescribe in this file)

Reply:
these were never running in the firs place so have asked the team to fix it , which will take some time

Reply:
is there a jira for that?

Reply:
@Neeraj-swarnkar is actively working on it